tests/sentry/integrations/github/test_integration.py

```diff
@@ -609,13 +609,11 @@ def get_installation_helper(self):
     def test_get_trees_for_org_works(self):
         """Fetch the tree representation of a repo"""
         installation = self.get_installation_helper()
+        cache.clear()
```

Comment:
I noticed that the next test would sometimes fail because the cache side effects of this test would affect the next.

I tried to add this as part of `tearDown` but it did not seem to matter. I will investigate on another occasion.